typescript/infra/config/environments/mainnet3/aw-validators/hyperlane.json

```diff
@@ -1,6 +1,8 @@
 {
   "ancient8": {
-    "validators": ["0xbb5842ae0e05215b53df4787a29144efb7e67551"]
+    "validators": [
```

Comment:
might need to `yarn prettier` - it's annoying but prettier fights the address serialization of the tooling